### DISCUSSION TITLE
Automation On Update Certain Column in Row

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#5393] [Help] [answered]
Hey, I'm trying to send out Emails once a Column changes its value which would be "Updated Row".
The Problem is that I need to be able to select a certain Column, not all of them.

Sincerely,
K

**Comment by melohagan** [comment, marked as answer]
Hi @kasgithub8 

Whenever "Updated Row" is triggered, it will be on a per row basis.

So you can access the row data for which the update occurred via: `trigger.row`

![Screenshot 2022-04-13 at 13 56 50](https://user-images.githubusercontent.com/101575380/163185061-f2b6d9af-99a2-4ec5-b83b-7918d81a0769.png)

**Reply by kasgithub8** [reply]
Hey @melohagan,

this is how it looks. 
![Screenshot 2022-04-14 075634](https://user-images.githubusercontent.com/95284278/163322879-6a5cded9-4089-4aba-b2af-a102e7389678.png)

Is it possible to choose a certain column here?

**Reply by melohagan** [reply]
@kasgithub8 Is there anything showing when you expand the *Setup* carrot for the Send Email step?

**Reply by kasgithub8** [reply]
@melohagan well, the Email Content is in there. 
What I want is: We have a contract. When 3 people approve it, a field "AllApproved" changes from "No" to "Yes" and I just want to send that  1 column update. "Row Updated" is just not precise enough.

**Reply by melohagan** [reply]
You could try adding an additional field "EmailSent" (Yes/No)

Then you would have a automation flow as follows:
1. Row Updated
2. Condition | *{{ trigger.row.AllApproved }}* Equals *Yes*
3. Condition | *{{ trigger.row.EmailSent }}* Equals *No*
4. Send Email
5. Update Row | Row ID = *{{ trigger.id }}*, EmailSent = *Yes*

**Reply by kasgithub8** [reply]
Damn, thanks for that 😄 
Works now!

**Reply by melohagan** [reply]
You're very welcome :)
